packages/yourloops/pages/signup/signup-profile-form.tsx

```diff
@@ -188,7 +188,7 @@ function SignUpProfileForm(props: SignUpFormProps): JSX.Element {
         id="lastname"
         className={classes.TextField}
         margin="normal"
-        label={t("lastname")}
+        label={t("lastName")}

```

Comment:
```suggestion
        label={t("lastname")}

```
Yourloops tr key is in lowercase